Gemfile

```diff
@@ -48,3 +48,5 @@ group(:development, :test) do
   # https://github.com/rails/rails/blob/0919aa97260ab8240150278d3b07a1547489e3fd/Gemfile#L178-L191
   gem("net-smtp", "0.3.1", require: false)
 end
+
+gem "rbi", github: "Shopify/rbi", branch: "main"
```

Comment:
I need a release of `rbi` so that I can bump the `.gemspec` dependency version instead of this. Until then this PR will be WIP.